README.md

```diff
@@ -179,44 +179,127 @@ Forks of this project demonstrate how to extend and integrate with other librari
 ├── package.json               <- dependencies of the project
 ├── protractor.conf.js         <- e2e tests configuration
 ├── src                        <- source code of the application
-│   ├── home
-│   │   └── components
-│   ├── index.html
-│   ├── main.ts
-│   ├── shared
-│   │   └── services
-│   │       ├── name-list...
-│   │       └── name-list...
-│   └── sw.js                  <- sample service worker
+│   └── client
+│       ├── app
+│       │   ├── +about
+│       │   │   ├── components
```

Comment:
@mgechev Shall i create a PR for that? Also another question: I know the `client` directory got introduced to enable multiple clients within the project. Yet, the seed does not (yet?) make use of that. I don't know if that'd be considered another violation to the styleguide, but i found this sample structure from the Styleguide (Style 04-06 Overall Structural Guidelines):

![screenshot 2016-05-11 16 36 15](https://cloud.githubusercontent.com/assets/1188033/15184710/9a4102b4-1796-11e6-98f2-e168a094c4d8.png)

So, should we keep the `client` subfolder?

Thank you for you input!